<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandBrowserToolboxWidget</class>
 <widget class="QDialog" name="StandBrowserToolboxWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandBrowser Toolbox</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabQField">
    <attribute name="title">
     <string>QField</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabGrid">
    <attribute name="title">
     <string>Grid</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>371</width>
       <height>191</height>
      </rect>
     </property>
     <property name="plainText">
      <string>Description:
Select layer from drop down
Grid spacing options:
1) Regular over each
2) Random over each
(Check box for treating selcted stand as one when it comes to spacing.

Create new layer or add to existing.

Use a template layer with fields from standid, but with points, or add to existing.

Can we use code from standard toolbox?</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbLayer">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use selected stand from layer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select distribution algorithm:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>290</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of points/stand</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbMinPoint">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>290</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>290</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>290</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>290</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbTemplate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string> Create new layer from template</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbTemplate">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>350</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbExisting">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to existing layer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTemplate">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="bbDialog">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
